core/blockchain/abci/tm_node.go

```diff
@@ -143,8 +143,6 @@ func overwriteConfig(config *config.Config) {
 	config.Consensus.CreateEmptyBlocks = true
 	// enforce using priority mempool
 	config.Mempool.Version = "v1"
-	// ensure rechecking tx is enabled
-	config.Mempool.Recheck = true
```

Comment:
Do we not still need this so that the checks in the PoW engine happen after every block? @ze97286

Reply:
yea we do.

Reply:
Yes, got removed in the revert, and I've added it back in the last commit I pushed :)